tests/make/test-common.mk

```diff
@@ -335,70 +364,57 @@ check-cleaned-blif: route.echo
 	$(YOSYS) -p "$(EQUIV_CHECK_SCRIPT)" $(TOP_DIR)/env/conda/share/yosys/$(CELLS_SIM) $(OUT_LOCAL)/atom_netlist.cleaned.echo.yosys.blif
 .PHONY: check-cleaned-blif
 
-check-post-blif: analysis.echo
+check-post-blif: $(OUT_LOCAL)/top_post_synthesis.blif
 	$(YOSYS) -p "$(EQUIV_CHECK_SCRIPT)" $(TOP_DIR)/env/conda/share/yosys/$(CELLS_SIM) $(OUT_LOCAL)/top_post_synthesis.blif
 .PHONY: check-post-blif
 
-check-post-v: analysis.echo
+check-post-v: $(OUT_LOCAL)/top_post_synthesis.v
 	$(YOSYS) -p "$(EQUIV_CHECK_SCRIPT)" $(TOP_DIR)/env/conda/share/yosys/$(CELLS_SIM) $(TOP_DIR)/vpr/primitives.v $(OUT_LOCAL)/top_post_synthesis.v
 .PHONY: check-post-v
 
-# Simulation
+# Compare the bitstream Verilog to original Verilog
+#-------------------------------------------------------------------------
+
+# Fix up names inside VCD files that gtkwave doesn't like
 $(OUT_LOCAL)/%.fixed.vcd: $(OUT_LOCAL)/%.vcd
-	cat $< | sed -e's/:/_/g' > $@
+	test -f $< && cat $< | sed -e's/:/_/g' > $@
```

Comment:
I think this sed was breaking some of the vcd files? Can we disable for now?

Reply:
Can you remember a case?

Reply:
I think it was the ram test benches....

Reply:
Ok I see the "fixed" signals now show as _counter[30:30]_. Do you know why we need to sed to fixed names in the first place?

Reply:
I think it had something to do with the autosim output. Can't remember...